Makefile

```diff
@@ -218,6 +276,7 @@ docker-build-cross-osm: docker-build-osm
 docker-build-cross-demo: DOCKER_BUILDX_PLATFORM=linux/amd64,windows/amd64
 docker-build-cross-demo: docker-build-demo
 docker-build-cross: docker-build-cross-osm docker-build-cross-demo
+docker-build-fips: docker-build-osm-fips docker-build-demo-fips
```

Comment:
If we want to run / test FIPS complaint images on ARM64, we may also want to consider a cross-build option in the makefile here.